### PR TITLE
added new logging library

### PR DESCRIPTION
**Pull request by anoop-b**
- Added [logrus](github.com/sirupsen/logrus) library for logging.  
- Logging files to terminal that are in queue to be sent.

If this logic looks good,
- The existing /logger folder can be removed entirely and only use logrus.
- create  a custom logger using logrus and replace all instances of existing logger across all files.

@claudiodangelis Awaiting for your thoughts on this.

### CODE REVIEW

**Comment by claudiodangelis** [comment]
Many thanks for this @anoop-b, I've been wanting to improve logging for a long time.
Please allow me a few days to review it, I'm particularly busy lately.
Cheers,
C

File: cmd/send.go [hunk 1]
```diff
@@ -2,19 +2,19 @@ package cmd
...
 	"github.com/claudiodangelis/qrcp/server"
 	"github.com/spf13/cobra"
 )
 
 func sendCmdFunc(command *cobra.Command, args []string) error {
-	log := logger.New(quietFlag)
+	// log := logger.New(quietFlag)
 	payload, err := payload.FromArgs(args, zipFlag)
 	if err != nil {
-		return err
```
Comment (claudiodangelis) [inline cmd/send.go:17]: I would prefer errors being returned rather than having the execution aborted.

File: cmd/send.go [hunk 2]
```diff
@@ -2,19 +2,19 @@ package cmd
...
 	"github.com/claudiodangelis/qrcp/qr"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/claudiodangelis/qrcp/server"
 	"github.com/spf13/cobra"
 )
 
 func sendCmdFunc(command *cobra.Command, args []string) error {
-	log := logger.New(quietFlag)
+	// log := logger.New(quietFlag)
```
Comment (claudiodangelis) [inline cmd/send.go:14]: Let's remove commented out code, it pollutes code search results!

File: cmd/send.go (hunk 1 above)
Comment (anoop-b) [reply cmd/send.go:17]: Isnt that what `log.Fatal(err)` does? to quote from the documentation, 
> Fatal logs a message at level Fatal on the standard logger then the process will exit with status set to 1.

File: cmd/send.go (hunk 2 above)
Comment (anoop-b) [reply cmd/send.go:14]: > Thanks for this contribution, the `github.com/claudiodangelis/qrcp/logger` can be removed altogether.

there's still references to the old logger in the `receive.go` , will replace them with the new logger .

File: cmd/send.go (hunk 1 above)
Comment (claudiodangelis) [reply cmd/send.go:17]: I'm saying the opposite of that: don't let the function exit the process, but rather return the error and let the caller function handle it. Does it make sense?

Comment (anoop-b) [reply cmd/send.go:17]: Ah, got it. Makes sense

**Pull request closed without merging**